contracts/modules/capital/Ramm.sol

```diff
@@ -417,6 +418,35 @@ contract Ramm is IRamm, MasterAwareV2 {
     );
   }
 
+  function getInitialObservations(
+    State memory initialState,
+    uint timestamp
+  ) public pure returns (Observation[3] memory initialObservations) {
+
+    Observation memory previousObservation;
+    uint endIdx = timestamp.divCeil(PERIOD_SIZE);
+
+    for (uint idx = endIdx - 2; idx <= endIdx; idx++) {
+      uint priceCumulativeAbove = previousObservation.priceCumulativeAbove;
+      uint priceCumulativeBelow = previousObservation.priceCumulativeBelow;
+
+      uint observationTimestamp = Math.min(timestamp, idx * PERIOD_SIZE);
+      uint observationIndex = idx % GRANULARITY;
+      uint timeElapsed = endIdx - 2 == idx ? INITIAL_PERIODS * PERIOD_SIZE : PERIOD_SIZE;
```

Comment:
Why are the first two using `INITIAL_PERIODS * PERIOD_SIZE` and the last one just `PERIOD_SIZE` for the time elapsed? `timeElapsed` doesn't make sense to me, maybe we can jump on a call in the morning.